src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Hover/HoverDisplayOptions.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Hover;
+
+internal readonly record struct HoverDisplayOptions(MarkupKind MarkupKind, bool SupportsVisualStudioExtensions)
```

Comment:
More options classes FTW!

If I was in a bikeshedding mood, I might ponder whether "Display" should be in the name because a) there are no other types of hover options, and b) we don't display anything 😛

Reply:
It is `HoverDisplayOptions` because `HoverOptions` already exists in the language server protocol itself, making for some very awkward type aliases. "Display" is fine. I'll draw your attention to Roslyn's famous "ToDisplayString()" which produces data "for display" just as we do for hover.

Reply:
Argh, of course. I missed that.

I would have probably called this `RazorHoverOptions` in that case, but just to be clear: This is fine, no need to change it.

Reply:
> Argh, of course. I missed that.
> 
> I would have probably called this `RazorHoverOptions` in that case, but just to be clear: This is fine, no need to change it.

Don't worry. I wasn't gonna. :smile: